merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/generator/MissionModelGenerator.java

```diff
@@ -586,6 +513,13 @@ private record ComputedAttributesCodeBlocks(TypeName typeName, FieldSpec fieldDe
     return generateCommonMapperMethods(missionModel, activityType).map(typeSpec -> typeSpec.toBuilder()
         .addMethod(makeGetReturnValueSchemaMethod())
         .addMethod(makeSerializeReturnValueMethod(activityType))
+        .addMethod(
+            MethodSpec
+                .methodBuilder("getName")
```

Comment:
> Would it be possible to add an `@Override` to this `getName` method? It does override the ActivityMapper interface after all.

Ah, good point. Yes, that would be possible.

Reply:
> I am curious why this method is necessary at all, since it's only used from generated code, and the name of the activity is known statically. I suppose it's good to have a complier-checkable "single source of truth" rather than generating a lot of string literals...

Only used from generated code, but might have to be implemented by user code. We allow users to define custom activity mappers, and I'd like them to be able to code to an interface rather than having to know somehow that there also needs to be a magical `public static final String NAME`.

Reply:
Amusingly, `getName` will drop out entirely in #86, so if you'd like we can just remove these changes and let #86 deal with `getName` more holistically.

Reply:
> and the name of the activity is known statically.

Ohh, I could just do the thing I did with the `Map` in `ActivityTypes` to statically use the `name` associated with the activity type. That's totally true, and it would be independent of whether an activty type has a user-defined mapper. Well, see above; I think shifting this change to #86 will be more holistic anyway!